notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/NotificationHub.java

```diff
@@ -0,0 +1,141 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+import android.app.Activity;
+import android.content.Intent;
+import com.google.firebase.messaging.RemoteMessage;
+import com.microsoft.windowsazure.messaging.notificationhubs.async.NotificationHubFuture;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ListIterator;
+
+/**
+ * A Singleton controller that wraps all interactions with Firebase Cloud Messaging and Azure
+ * Notification Hubs.
+ */
+public final class NotificationHub {
+    private static NotificationHub sInstance;
+
+    private NotificationListener mListener;
+    private final List<InstallationMiddleware> mMiddleware;
+    private final PushChannelEnricher pushChannelEnricher;
+    private InstallationManager manager;

```

Comment:
Mix of `m` and non `m` prefixed variables.  Let's stick with prepending `m`

Reply:
Fixed in 583cdf30d92d7f9a3962e8b37fae21d9037b4511